corehq/apps/cloudcare/views.py

```diff
@@ -157,6 +157,22 @@ def set_cookie(response):  # overwrite the default noop set_cookie
                     response.delete_cookie(cookie_name)
                     return response
 
+        elif request.couch_user.has_permission(domain, 'limited_login_as'):
+            login_as_users = login_as_user_query(
+                domain,
+                request.couch_user,
+                search_string='',
+                limit=1,
+                offset=0
+            ).run()
+            if login_as_users.total == 1:
```

Comment:
If `limit=1` won't it always return at most one user? Which is probably fine, since I don't think the custom user property supports multiple mobile workers, it's just strange to read this code without that context.

Reply:
Not quite. It could return no users, if none have been set up for this web user. But total is a count of all matches, not just what is returned, so if there are more than 1 mobile workers set up for the web user, only 1 doc will come back in `hits` but `total` will have the full number.

Reply:
Yes, at most one. Didn't realize total will have the real total, sounds good.